kraken-app/kraken-app-portal/src/utils/helpers/validators.ts

```diff
@@ -3,9 +3,15 @@ import { isEmpty } from 'lodash';
 import { IProductIdAndNameParams } from '../types/product.type';
 import { isURL } from './url';
 
-export const validateServerName = async (validateName: UseMutateAsyncFunction<any, Error, IProductIdAndNameParams, unknown>, currentProduct: string, name: string) => {
+export const validateServerName = async (
+  validateName: UseMutateAsyncFunction<any, Error, IProductIdAndNameParams, unknown>,
+  currentProduct: string,
+  name: string,
+  originalName: string
+) => {
+
   const { data: isValid } = await validateName({ productId: currentProduct, name });
```

Comment:
`name = originalName` can be a base case. It should be lifted up before row 13
```suggestion
  if (name === originalName) return Promise.resolve()
  const { data: isValid } = await validateName({ productId: currentProduct, name });
  ...
```